<launch>
    <arg name="joy_config" default="mallard_ps4" />
    <arg name="joy_dev" default="/dev/input/js1" />
    <arg name="joy_dev_ff" default="/dev/input/event1" />

    <group ns="mallard">
        <node pkg="joy" type="joy_node" name="joy_node">
            <param name="dev" value="$(arg joy_dev)" />
            <param name="deadzone" value="0.3" />
            <param name="autorepeat_rate" value="20" />
            <param name="default_trig_val" value="true" />
            <param name="dev_ff" value="$(arg joy_dev_ff)" />
            <param name="coalesce_interval" type="double" value="0.05" />
        </node>

        <node pkg="taarn_teleop_joy" name="teleop_joy" type="teleop_node" output="screen">
            <rosparam command="load" file="$(find taarn_teleop_joy)/config/$(arg joy_config).config.yaml" />
            <remap from="cmd_vel" to="teleop/cmd_vel"/>
        </node>
    </group>
</launch>
